### DISCUSSION TITLE
Problem with API token

### DISCUSSION THREAD

**Discussion by bengtove** [jamiebrynes7/obsidian-todoist-plugin#335] [General]
I'm trying to install Todoist Sync, but I got stuck after I supplied the API token. Submitting the token is impossible — the submit button doesn't work – and hitting return has no effect. I've created a new token, turned the plugin on and off, and restarted Obsidian, but this hasn't helped.

Any advice? If it makes any difference, I'm running Obsidian on a Mac.

**Comment by susangracetaylor** [comment]
Same issue here; I've switched from one ToDoist account to another and need to update the API token. Once you input a token, is it not able to be changed? Also running Obsidian on Mac. 
<img width="381" alt="Screenshot 2024-10-17 at 8 55 51 AM" src="https://github.com/user-attachments/assets/ee5139be-eeec-4fc8-bd20-29d19a3f55a4">

**Reply by jamiebrynes7** [reply]
To remove the token, delete the `todoist-token` file from inside the `.obsidian` directory in your vault. I should probably add a button to the settings to do this 😅
